### ISSUE TITLE
Publishing Clean Arhitecture Project with Angular and .net core in Visual Studio

### ISSUE THREAD

**Issue by aleksflorin** [jasontaylordev/CleanArchitecture#343]
**Describe the bug**
I was trying to publish a solution using Visual Studio but I'm really stucked at this point. I got my little project using Clean arhitecture solution template but when trying to publish it nothing seems to work.

**To Reproduce**
Publishing the WebUi project

**Expected behavior**
To publish succesfully

**Screenshots**
[Screenshot here](https://i.stack.imgur.com/uDilN.png)

**Additional context**
What I've tried:

-publishing each project one by one, but the frontend (angular) won't communicate with the backend(.net core)
-updating node and python
-reinstalling python and node
-running the visual studio by administrator
-trying to publish the template from github, without my very own code in it
-update the csproj file from npm run build -- --prod to npm run build --prod
I also tried to look for this error on the internet, I already tried every solution but nothing seems to work for me.

**Comment by ZVARKO** [comment]
I installed latest template version (1.1.4) via dotnet CLI, created new project from the template, tried to publish it via Visual Studio and also encountered this issue. I was able to fix it by removing optionalDependencies from [package.json](https://github.com/jasontaylordev/CleanArchitecture/blob/d0f133ee026aec5cd5856c5592c307b5f20fa8e4/src/WebUI/ClientApp/package.json#L57). Just remove these 6 lines and it should work.

`"optionalDependencies":{
"node-sass": "^4.13.1",
  "protractor": "~7.0.0",
  "ts-node": "~5.0.1",
  "tslint": "~6.1.0"
}`
